### ISSUE TITLE
Add alt attribute to previous and next buttons to pass a11y audit in Lighthouse

### ISSUE THREAD

**Issue by alejost848** [FabricElements/skeleton-carousel#87]
Right now the element is giving this alert on the Accessibility section from a Lighthouse audit.

![image](https://user-images.githubusercontent.com/8323249/43929118-2d2178aa-9bf9-11e8-9ea2-8e58f35248ec.png)

I'm no a11y expert, but I think it should be as easy as adding `alt="Previous slide (or) Next slide"` to each icon button.
